packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -175,6 +175,19 @@ class ErrorHint extends _ErrorDiagnostic {
   ErrorHint._fromParts(List<Object> messageParts) : super._fromParts(messageParts, level:DiagnosticLevel.hint);
 }
 
+/// An [ErrorSpacer] creates an empty [DiagnosticsNode], that can be used tune
```

Comment:
nit: there seems to be a word missing? "can be used *to* tune"?

Reply:
done.